### PR TITLE
Feature/chips component

### PR DESCRIPTION
**Pull request by joohncruz**
Added chip and input-chip components. 

I recommend pulling this branch out and checking the component in the storybook.

### CODE REVIEW

File: src/components/chip/test/chip.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+describe('bds-chip', () => {
+  let page;
+
+  beforeEach(async () => {
+    page = await newSpecPage({
+      components: [Chip],
+      html: `<bds-chip></bds-chip>`,
+    });
+  });
+
```
Comment (vinioo) [inline src/components/chip/test/chip.spec.tsx:13]: add tests to props and the event emitter

File: src/components/input-chips/tests/input-chips.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('bds-input-chips', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<bds-input-chips></bds-input-chips>');
+
```
Comment (vinioo) [inline src/components/input-chips/tests/input-chips.e2e.ts:7]: add tests to props, events and methods

File: src/components/input/input.tsx [hunk 3]
```diff
@@ -1,5 +1,7 @@
+/* eslint-disable no-console */
```
Comment (vinioo) [inline src/components/input/input.tsx:1]: remove it

File: src/components/chip/chip.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  decorators: [withKnobs],
+  parameters: {
+    notes: { markdown: readme },
+  },
+};
+
+export const chipDefault = () => {
+  const sizeOptions = select('size', ['standard', 'tall']);
+  const variantOptions = select('variant', ['default', 'primary']);
+  const slot = text('text', 'rafael@take.net');
```
Comment (Farrael22) [inline src/components/chip/chip.stories.jsx:17]: Change it to jhonny@take.net

Comment (Farrael22) [inline src/components/chip/chip.stories.jsx:17]: Change it to jhonny@take.net

Comment (joohncruz) [reply src/components/chip/chip.stories.jsx:17]: nop

File: src/components/chip/test/chip.spec.tsx (hunk 1 above)
Comment (joohncruz) [reply src/components/chip/test/chip.spec.tsx:13]: fix

File: src/components/input-chips/tests/input-chips.e2e.ts [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  it('should add chips type email before fill value and press Enter (writing email)', async () => {
+    const page = await newE2EPage({
+      html: `<bds-input-chips type="email"></bds-input-chips>`,
+    });
+
+    const inputRootElement = await page.find('bds-input-chips');
+    const inputNativeElement = await page.find('bds-input-chips >>> input');
+
+    await inputNativeElement.press('b');
```
Comment (vinioo) [inline src/components/input-chips/tests/input-chips.e2e.ts:70]: in this case you can use

`inputNativeElement.type('email@.com'); `

and then

`await inputRootElement.press('Enter');`

**Pull request merged**